S3/English/ASS_32.ass

```diff
@@ -0,0 +1,102 @@
+﻿[Script Info]
+; Script generated by Aegisub 3.2.2
+; http://www.aegisub.org/
+ScriptType: v4.00+
+WrapStyle: 0
+ScaledBorderAndShadow: yes
+YCbCr Matrix: TV.601
+PlayResX: 1920
+PlayResY: 1080
+
+[Aegisub Project Garbage]
+Audio File: 万圣街_1920x1080_万圣街_32.mp4
+Video File: 万圣街_1920x1080_万圣街_32.mp4
+Video AR Mode: 4
+Video AR Value: 1.777778
+Video Zoom Percent: 0.375000
+Active Line: 19
+Video Position: 2872
+
+[V4+ Styles]
+Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
+Style: Default,Arial,45,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,18,0,20,1
+Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,10,10,280,1
+
+[Events]
+Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
+Dialogue: 0,0:01:02.51,0:01:05.56,Title,,0,0,0,, {\pos(970,920)} {\b1\an5\pos(970,880)\fs120\frz4.5\t(0,100,1.5\fs45)}Episode 32:  {\c&H7990E7&}How much surprise that I didn't know{\b0}
+Dialogue: 0,0:01:15.13,0:01:16.70,Default,,0,0,0,,Oh, why didn't Ira come
+Dialogue: 0,0:01:17.30,0:01:18.70,Default,,0,0,0,,He'll be damaged by the sun
+Dialogue: 0,0:01:19.93,0:01:20.45,Default,,0,0,0,,Next
+Dialogue: 0,0:01:22.19,0:01:23.51,Default,,0,0,0,,I'll be with Lily next
+Dialogue: 0,0:01:24.04,0:01:24.95,Default,,0,0,0,,Should I sit in the front
+Dialogue: 0,0:01:25.54,0:01:26.82,Default,,0,0,0,,Don't be afraid, hold me close
+Dialogue: 0,0:01:27.00,0:01:27.50,Default,,0,0,0,,Neil
+Dialogue: 0,0:01:27.78,0:01:29.10,Default,,0,0,0,,Or should I sit in the back
+Dialogue: 0,0:01:29.10,0:01:30.25,Default,,0,0,0,,Don't worry, I'm here
+Dialogue: 0,0:01:30.81,0:01:31.33,Default,,0,0,0,,Neil
+Dialogue: 0,0:01:32.42,0:01:33.69,Default,,0,0,0,,So hard to decide
+Dialogue: 0,0:01:33.97,0:01:34.79,Default,,0,0,0,,Neil
+Dialogue: 0,0:01:35.25,0:01:36.11,Default,,0,0,0,,We're next
+Dialogue: 0,0:01:37.97,0:01:38.90,Default,,0,0,0,,Let's go
+Dialogue: 0,0:01:42.04,0:01:43.64,Default,,0,0,0,,Next
+Dialogue: 0,0:01:54.39,0:01:55.09,Default,,0,0,0,,It's awesome!
+Dialogue: 0,0:01:55.26,0:01:56.75,Default,,0,0,0,,Let's do this again. Luis
+Dialogue: 0,0:01:58.39,0:01:58.80,Default,,0,0,0,,Lulu
+Dialogue: 0,0:01:59.64,0:02:01.60,Default,,0,0,0,,Somebody help! Somebody died
```

Comment:
Need better tl

Reply:
it actually fits the situation so maybe ask Tori instead.